### PR TITLE
Feature/steps slider

### PR DESCRIPTION
**Pull request by j8seangel**
- Adds the navigation slider. By default, the swipe it is locked and only can navigate with the back/next button.
- Adds new header component.
- Saves the user country and validate it.
- Creates next button component to reuse styles and behaviour (now you can disable if you need).
- Includes empty overview setup component.

image

### CODE REVIEW

**Comment by j8seangel** [comment]
The next button was moved to common to make it reusable, now accepts this props:
- style: custom styles, normally used to positioned it
- light: white background
- disabled: grey background with onPress action disabled

File: app/components/common/action-button/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+function NextButton(props) {
+  function onButtonPress() {
+    if (!props.disabled) {
+      props.onPress();
+    }
+  }
+  const btnStyles = [
+    styles.container,
+    props.light ? styles.light : '',
+    props.disabled ? styles.disabled : '',
```
Comment (geriux) [inline app/components/common/action-button/index.js]: Niceee!

File: package.json [hunk 2]
```diff
@@ -9,10 +9,10 @@
   "dependencies": {
     "react": "15.4.1",
     "react-native": "0.38.0",
-    "react-native-app-intro": "^1.1.5",
```
Comment (geriux) [inline package.json]: 👋

File: package.json [hunk 3]
```diff
@@ -9,10 +9,10 @@
...
     "react-native-config": "^0.2.1",
     "react-native-i18n": "^0.1.1",
     "react-native-maps": "^0.12.2",
+    "react-native-scrollable-tab-view": "^0.7.2",
```
Comment (geriux) [inline package.json]: 🤗

File: app/main.js [hunk 4]
```diff
@@ -19,19 +19,32 @@ const store = createStore(
...
   )
 );
 
-// Add `autoRehydrate()`  in the compose once redux-persist works on Android
+// Disable ios warnings
+// console.disableYellowBox = true;
 
+const persistConfig = {
+  storage: AsyncStorage,
+  blacklist: ['navigation']
```
Comment (geriux) [inline app/main.js]: Perfect!!!

**Comment by j8seangel** [comment]
Instead of purge the store once loaded I've added the navigation reducer in the blacklist of the redux-persist to avoid save its status.

**Comment by geriux** [comment]
Awesome work!!!!

**Pull request merged**